website/public/img/logo-text.svg

```diff
@@ -1,8 +1,91 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 342 107">
-  <g fill="none" fill-rule="evenodd">
-    <path class="text" fill="#000000" fill-rule="nonzero" d="M128.8 40c0-10.8 6.3-17 20.9-17 5.5 0 11.1.7 16 1.9l-1.2 9.6c-4.8-.9-10.3-1.5-14.4-1.5-7.7 0-10.1 2.6-10.1 8.9v22.6c0 6.3 2.5 8.9 10.1 8.9 4.1 0 9.6-.6 14.4-1.5l1.2 9.6c-4.8 1.2-10.5 1.9-16 1.9-14.6 0-20.9-6.3-20.9-17V40zm58.8 43.5c-14.7 0-18.7-8.1-18.7-16.9V55.8c0-8.8 4-16.9 18.7-16.9s18.7 8.1 18.7 16.9v10.8c0 8.8-3.9 16.9-18.7 16.9zm0-35.6c-5.7 0-7.9 2.6-7.9 7.4v11.5c0 4.8 2.2 7.4 7.9 7.4s7.9-2.6 7.9-7.4V55.3c.1-4.8-2.1-7.4-7.9-7.4zm49.7 34.7v-30c0-2.3-1-3.4-3.4-3.4-2.6 0-7.3 1.6-11.2 3.6v29.8h-10.8V39.7h8.2l1.1 3.6c5.4-2.6 12.2-4.5 17.2-4.5 7.1 0 9.7 5 9.7 12.7v31.1h-10.8zm31.1.9c-4.6 0-10.8-1.1-14.8-2.5l1.5-8.2c3.6 1.1 8.4 1.9 12.8 1.9 4.8 0 5.5-1.1 5.5-4.3 0-2.6-.5-4-7.5-5.6-10.5-2.6-11.7-5.2-11.7-13.5 0-8.6 3.8-12.4 16-12.4 4 0 9.2.5 12.9 1.5L282 49c-3.3-.6-8.5-1.2-11.8-1.2-4.7 0-5.5 1.1-5.5 3.7 0 3.4.3 3.7 6.1 5.2 12 3.2 13.1 4.8 13.1 13.6 0 8.1-2.5 13.2-15.5 13.2zM300 39.7v30c0 2.3 1 3.4 3.4 3.4 2.6 0 7.3-1.6 11.2-3.6V39.7h10.8v42.9h-8.2l-1.1-3.6c-5.4 2.6-12.2 4.5-17.2 4.5-7.1 0-9.7-5-9.7-12.7V39.7H300zm31.1 42.9V22.1l10.8-1.5v62"/>
-    <path class="center-c" fill="#8C1C59" d="M53.2 64.9c-6.3 0-11.4-5.1-11.4-11.4 0-6.3 5.1-11.4 11.4-11.4 6.3 0 11.4 5.1 11.4 11.4 0 6.3-5.1 11.4-11.4 11.4"/>
-    <path class="circles" fill="#C62A71" d="M75.4 58.8c-2.9 0-5.3-2.3-5.3-5.3 0-2.9 2.3-5.3 5.3-5.3 2.9 0 5.3 2.3 5.3 5.3 0 2.9-2.4 5.3-5.3 5.3m19.2 4.9c-.7 2.8-3.6 4.5-6.4 3.8-2.8-.7-4.5-3.6-3.8-6.4.7-2.8 3.6-4.5 6.4-3.8 2.7.7 4.4 3.3 3.9 6 0 .2 0 .3-.1.4m-3.7-13.5c-2.8.7-5.7-1.1-6.3-3.9-.6-2.8 1.1-5.7 3.9-6.3 2.8-.7 5.7 1.1 6.3 3.9.1.7.2 1.4.1 2.1-.3 2-1.8 3.7-4 4.2m18.7 13c-.5 2.9-3.2 4.8-6.1 4.3-2.9-.5-4.8-3.2-4.3-6.1.5-2.9 3.2-4.8 6.1-4.3 2.7.5 4.6 2.9 4.4 5.6 0 .1-.1.3-.1.5M105.3 50c-2.9.5-5.6-1.5-6-4.3-.5-2.9 1.5-5.6 4.3-6 2.9-.5 5.6 1.5 6 4.3.1.5.1.9.1 1.4-.2 2.2-2 4.2-4.4 4.6m-3.6 31.4c-1.5 2.5-4.6 3.4-7.2 2-2.5-1.5-3.4-4.6-2-7.2 1.5-2.5 4.6-3.4 7.2-2 1.8 1 2.8 3 2.6 5 0 .8-.2 1.5-.6 2.2M99.8 33c-2.5 1.4-5.7.5-7.2-2-1.4-2.5-.5-5.7 2-7.2 2.5-1.4 5.7-.5 7.2 2 .6 1 .8 2 .6 3-.1 1.8-1.1 3.4-2.6 4.2"/>
-    <path class="c" fill="#C62A71" fill-rule="nonzero" d="M53.4 106.5C39.2 106.5 26 101 16 91S.5 67.7.5 53.6C.5 39.5 6 26.2 16 16.2S39.3.7 53.4.7c11.7 0 22.8 3.8 32.1 10.9L79 20c-7.4-5.7-16.3-8.7-25.6-8.7-11.3 0-21.9 4.4-29.9 12.4S11.2 42.3 11.2 53.6c0 11.3 4.4 21.9 12.4 29.9s18.6 12.3 29.9 12.3c9.4 0 18.2-3 25.6-8.7l6.5 8.4c-9.4 7.3-20.5 11-32.2 11z"/>
-  </g>
```

Comment:
Why was this file updated?

Reply:
Because it was the old logo: https://github.com/hashicorp/consul/blob/main/website/public/img/logo-text.svg

I replaced it with the equivalent new logo: https://github.com/hashicorp/consul/blob/badge-improvements/website/public/img/logo-text.svg

I thought it made sense to update both the logos (with and without `HashiCorp`) at the same time, since I was referencing one of them in this README